pkg/kubectl/cmd/run_test.go

```diff
@@ -232,8 +231,8 @@ func TestGenerateService(t *testing.T) {
 					Selector: map[string]string{
 						"run": "foo",
 					},
-					Type:            api.ServiceTypeClusterIP,
-					SessionAffinity: api.ServiceAffinityNone,
+					Type:            v1.ServiceTypeClusterIP,
+					SessionAffinity: v1.ServiceAffinityNone,
```

Comment:
the unit test failure is legitimate... previously, this was going through defaulting. now, it's not, so Type:ClusterIP and SessionAffinity:None are no longer set

Reply:
got it, updated